source/linear-algebra/source/01-LE/02.ptx

```diff
@@ -644,64 +644,82 @@ in many applications we use it for in <xref ref="EV"/>.
+            </li>
+            <li> Add 2 row 3 to row 2 (<m>R_2+2R_3 \rightarrow R_2</m>)
+            </li>
+        </ol>
+    </task>
 
-<activity estimated-time='7'>
-    <statement>
+    <task>
         <p>
-Consider the matrix
-<me>\left[\begin{array}{cccc} \markedPivot{1} &amp; 3 &amp; -1 &amp; 2 \\ 2 &amp; 6 &amp; -1 &amp; 6 \\ -1 &amp; -3 &amp; 2 &amp; 0 \end{array}\right].</me>
-Which row operation is the best choice for the next move in converting to RREF?
+            Consider the matrix:
+            <me>\left[\begin{array}{cccc}\markedPivot{1} &amp; 3 &amp; -1 &amp; 2  \\ 0 &amp; 0 &amp; 1 &amp; 2 \\ 0 &amp; 0 &amp; 1 &amp; 2 \end{array}\right].</me>
+            Which row operation is the best choice for the next move in converting to RREF?
         </p>
-<ol marker="A.">
-<li> Add row 1 to  row 3 (<m>R_3+R_1 \rightarrow R_3</m>)
-</li>
-<li> Add -2 row 1 to  row 2 (<m>R_2-2R_1 \rightarrow R_2</m>)
-</li>
-<li> Add 2 row 2 to row 3 (<m>R_3+2R_2 \rightarrow R_3</m>)
-</li>
-<li> Add 2 row 3 to row 2 (<m>R_2+2R_3 \rightarrow R_2</m>)
-</li>
-</ol>
-    </statement>
-</activity>
+        <ol marker="A.">
+            <li> Add row 1 to row 2 (<m>R_2+R_1 \rightarrow R_2</m>)
+            </li>
+            <li> Add -1 row 3 to  row 2 (<m>R_2-R_3 \rightarrow R_2</m>)
+            </li>
+            <li> Add -1 row 2 to  row 3 (<m>R_3-R_2 \rightarrow R_3</m>)
+            </li>
+            <li> Add row 2 to row 1 (<m>R_1+R_2 \rightarrow R_1</m>)
+            </li>
+        </ol>
+    </task>
 
-<activity estimated-time='5'>
-    <statement>
+    <task>
         <p>
-Consider the matrix
-<me>\left[\begin{array}{cccc}\markedPivot{1} &amp; 3 &amp; -1 &amp; 2  \\ 0 &amp; 0 &amp; 1 &amp; 2 \\ 0 &amp; 0 &amp; 1 &amp; 2 \end{array}\right].</me>
-Which row operation is the best choice for the next move in converting to RREF?
+            Consider the matrix:
+            <me>\left[\begin{array}{cccc}2 &amp; 6 &amp; -1 &amp; 6  \\ 1 &amp; 3 &amp; -1 &amp; 2 \\ -1 &amp; -3 &amp; 2 &amp; 0 \end{array}\right].</me>
+            Mark the position where the first pivot should be. 
+            Which row operation is the best choice for the first move in converting to RREF?
         </p>
-<ol marker="A.">
-<li> Add row 1 to row 2 (<m>R_2+R_1 \rightarrow R_2</m>)
-</li>
-<li> Add -1 row 3 to  row 2 (<m>R_2-R_3 \rightarrow R_2</m>)
-</li>
-<li> Add -1 row 2 to  row 3 (<m>R_3-R_2 \rightarrow R_3</m>)
-</li>
-<li> Add row 2 to row 1 (<m>R_1+R_2 \rightarrow R_1</m>)
-</li>
-</ol>
-    </statement>
+        <ol marker="A.">
+            <li> Add row 3 to row 2 (<m>R_2+R_3 \rightarrow R_2</m>)
+            </li>
+            <li> Add row 2 to row 3 (<m>R_3+R_2 \rightarrow R_3</m>)
+            </li>
+            <li> Swap row 1 to row 2 (<m>R_1 \leftrightarrow R_2</m>)
+            </li>
+            <li> Add -2 row 2 to row 1 (<m>R_1-2R_2 \rightarrow R_1</m>)
+            </li>
+        </ol>
+    </task>
```

Comment:
This appears to be accidentally identical to the first task.

Reply:
oh wait well maybe I should read your PR description. Then again, instructors won't read the PR description either - I don't see how this repetition helps just by reading the activity book, but it's late and I should look again with fresh eyes in the morning.

Reply:
@siwelwerd can you clarify what you're proposing. I did a few things and I'm not clear if we're rejecting all of them. 

First I grouped the activities into one multi-part activity (do we like this or not?)

Secondly, are you suggesting adding a separate activity in which we revisit the first activity and then include an instructor note?

Reply:
I'm saying to reject what Steven proposed, but add an instructor note as to why it is repeated, as some instructors might think it's an error

Reply:
working on the note now.